### PR TITLE
Added .gitattributes. Fixed typo in README.

### PR DESCRIPTION
**Pull request by theodesp**

### CODE REVIEW

**Comment by theodesp** [comment]
https://github.com/pshirshov/izumi-r2/issues/344

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Key goals
 
 We aim to provide tools which are:
 
-1. Boosting productiviy 
+1. Boosting productivity while reducing code-bloat
```
Comment (neko-kai) [inline README.md:44]: What's the motivation? I feel like a more concise description is better.

Comment (theodesp) [reply README.md:44]: I thought a more convincing fact will be handy here.

Comment (neko-kai) [reply README.md:44]: Fair enough, though we tend to focus on pragmatic cost-benefit reasoning. :) "Boosting productivity while reducing maintenance $(cost|burden)" could fit better.

**Pull request merged**